### DISCUSSION TITLE
No longer in Google Play

### DISCUSSION THREAD

**Discussion by curtisdr** [thecowan/wallpanel-android#330] [Q&A]
Is this app going to return to the Google Play Store?  I sure hope so.

**Comment by TylerRudie** [comment]
Also the Releases is empty and Amazon says No longer being sold in the US... Did this project die?

**Reply by curtisdr** [reply]
- I read ownership was transfered, so hopefully this is just a delay.  I got it & side-loaded it on my android tab from [the release section.](https://github.com/thanksmister/wallpanel-android/releases)

**Reply by Chacsam** [reply]
following as I am gradually adopting the app
